### PR TITLE
Create separate containers for Java and Python

### PR DESCRIPTION
**Pull request by dinogun**
Update build and deploy scripts to support the same.
We will still have a single pod that will house both containers to enable the service connections to work the same.

### CODE REVIEW

File: Dockerfile.optuna [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2020, 2021 Red Hat, IBM Corporation and others.
```
Comment (chandrams) [inline Dockerfile.optuna:2]: Should 2020 be 2021?

File: deploy.sh [hunk 2]
```diff
@@ -121,8 +126,8 @@ do
 	n)
 		autotune_ns="${OPTARG}"
 		;;
-	d)
-		AUTOTUNE_CONFIGMAPS="${OPTARG}"
+	o)
+		OPTUNA_DOCKER_IMAGE="${OPTARG}"
```
Comment (chandrams) [inline deploy.sh:130]: Extra tab / space present

**Comment by chandrams** [comment]
@dinogun - Please fix the PR check

**Comment by chandrams** [comment]
Tried it manually, build worked but deploying autotune failed with CrashLoopBackOff on my minikube setup

**Comment by dinogun** [comment]
It looks like #299 was missing a dependency and the PR check did not catch it as the error got lost with the java messages. Infact this PR helped to point it out! Anyway it is now fixed by adding the `jsonschema` package in the pip install.

**Comment by dinogun** [comment]
Hmmm on closer look, #299 did add that dependency, but it got missed in my PR as it was not synced, my bad.

**Review by chandrams — APPROVED** [review]
LGTM

**Pull request merged**